devops/druid-provision.yml

```diff
@@ -0,0 +1,35 @@
+-
+  name: Install zookeeper cluster for druid
+  hosts: druid-zookeeper
+  become: yes
+  roles:
+    - {role: zookeeper-upgrade}
+  vars:
+    zookeeper_group: "{{ groups['druid-zookeeper'] }}"
+
+-
+  name: Install postgres on druid cluster
+  hosts: druid-postgres
+  become: yes
+  roles:
+    - {role: postgres-provision }
+  vars:
+    postgres_address_space: "0.0.0.0/0"
```

Comment:
Please provide a way to override the postgres_address_space so that we can restrict the machines from which the database can be accessed.

Reply:
This has already been converted to a variable and we are overriding the value here.